schemas/string_interpolation/transforms/email_link.json

```diff
@@ -0,0 +1,31 @@
+{
+  "$id": "https://eq.ons.gov.uk/string_interpolation/transforms/email_link.json",
+  "$schema": "http://json-schema.org/draft-07/schema",
+  "email_link": {
+    "type": "object",
+    "properties": {
+      "transform": {
+        "type": "string",
+        "enum": ["email_link"]
+      },
+      "arguments": {
+        "type": "object",
+        "properties": {
+          "email_address": {
+            "$ref": "https://eq.ons.gov.uk/common_definitions.json#/non_empty_string"
```

Comment:
Email address validation could be done in schema rather than code - https://json-schema.org/understanding-json-schema/reference/string.html#email-addresses

Reply:
Something like:
```
  "email": {
    "$ref": "https://eq.ons.gov.uk/common_definitions.json#/non_empty_string",
    "description": "",
    "pattern": "^[A-Z0-9._%+-]+[A-Z0-9.-]+\\.[A-Z]{2,}$"
  },
```

Reply:
Similar validation for telephone number...

Reply:
True but the card said to use the runner implementation which is why I used email-validator